packaging/nnstreamer.spec

```diff
@@ -772,7 +772,7 @@ cp -r result %{buildroot}%{_datadir}/nnstreamer/unittest/
 %files
 
 %files core
-%manifest nnstreamer.manifest
+%manifest %{name}.manifest
```

Comment:
I wonder if this should be updated in other locations as well like line 793, 800, etc?

Reply:
You are right. I will fix it if this patch is necessary.